packages/node-cli/src/commands/CreateProjectCommandGenerator.js

```diff
@@ -363,6 +338,28 @@ module.exports = class CreateProjectCommandGenerator extends BaseCommandGenerato
 		);
 	}
 
+	_sanitizeName(name) {
```

Comment:
Confirmed with Carlos. Let's use static name in template file. It's going to be "suitecloud-project", no need to differentiate between SuiteApp and ACP in the name